### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN021.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#67] [General]
# https://blog.3place.cn/3place/lmyn/LMYN021.html

https://blog.3place.cn/3place/lmyn/LMYN021.html

**Comment by picklecai** [comment]
十车王呀！如果你不做的话，你那祭祀功果就要丧失，因此，请把罗摩送走吧！

这句恐吓的话，是婆私吒说的。

**Comment by picklecai** [comment]
一切兵器都是俱利舍湿婆〔206〕的
那一些极端虔诚的儿子，
它们从前被送给了众友，
当时众友正把国家来统治〔207〕。 （1.20.13）

众友，曾经掌管天下兵器。

**Comment by picklecai** [comment]
行大苦行，有大功德。

努力，且有成就。

**Comment by picklecai** [comment]
婆私吒的游说：
1. 生在王族，你一直以智慧著称。
2. 你预先已经答应了，现在不同意就算反悔，就会前功尽弃。
3. 罗刹不是罗摩的对手。原因是众友会保护他。
4.众友娴熟各种兵器。原因是过去兵器之母曾经把所有的兵器都送给了当时还在当国王的众友。
5. 众友智慧很高，善于创造。
6. 总结：众友英勇，行大苦行，有大功德。

第一项和第二项，和众友的游说没区别。捧和吓并重。
从第三项开始，具体论述罗摩不会有问题的原因，全在众友身上。这样的众友，来保护罗摩，所以没问题。
潜台词就是：这样的众友，你如果得罪了，那你和你的国家将会遭的殃，也会更厉害。
即使没有这个潜台词，或者十车王没有听懂这个潜台词。大吹众友的实力，也确实有利于十车王放下忧虑。因为，十车王所有的力量加起来，也无法和众友的一丁点力量相提并论。

所以下文十车王马上愉快地献出了儿子。
